mobile/packages/swap/manager.ts

```diff
@@ -158,21 +167,12 @@ const apiManagerMaker = (
 
       /* istanbul ignore next */
```

Comment:
for packages only mocks shouldn't be tested code is a must 100% coverage

Reply:
I don't think it's possible to get 100%